### ISSUE TITLE
too many scroll bars / can scroll way too far

### ISSUE THREAD

**Issue by jsoref** [argoproj/argo-cd#4773]
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

**To Reproduce**

1. Visit https://cd.apps.argoproj.io/applications/cert-manager?node=apps%2FDeployment%2Fcert-manager%2Fcert-manager-webhook%2F0
1. Use trackpad to scroll down

**Expected behavior**

* The last line of the live manifest should be the last line of the live manifest
* The head of life manifest / diff / desired manifest shouldn't be able to scroll off the top of the screen
* While the pane is open, one shouldn't be scrolling the applications screen behind it

**Screenshots**

![image](https://user-images.githubusercontent.com/2119212/98323177-38f52c00-1fb7-11eb-870d-24cea52da0b2.png)
![image](https://user-images.githubusercontent.com/2119212/98323223-50ccb000-1fb7-11eb-8515-856f60f6d2a6.png)
![image](https://user-images.githubusercontent.com/2119212/98323393-adc86600-1fb7-11eb-8ed5-3f29b6fcad19.png)
![image](https://user-images.githubusercontent.com/2119212/98323442-c2a4f980-1fb7-11eb-9f70-b2d6815c6d39.png)

**Version**

```shell
{
    "Version": "v1.8.0+3e19b2f",
    "BuildDate": "2020-11-05T22:00:27Z",
    "GitCommit": "3e19b2fdf144bb924ecb92614f9ff4d5eb2b5d35",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.1",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.0+g7090a89",
    "KubectlVersion": "v1.17.8",
    "JsonnetVersion": "v0.16.0"
}
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by ruant** [comment]
Is there something internally with argo-cd or is monaco that is the issue?

**Comment by jsoref** [comment]
@ruant: Until proven otherwise, assume argo-cd is responsible for the behavior. It's trivially reproducible (my steps above worked today), give it a try and look at what things are doing...

**Comment by ruant** [comment]
@jsoref Yeah, I can reproduce it fine. And it's quite an annoying bug too. Since the "outer" scrollbar is in focus, the "inner" isn't scrolling and you gotta manually drag the scrollbar to view the bottom part of the live manifest.  

I did poke around a little to try to figure out if it was some easy CSS fix. Seams to be some JS update of the height property from monaco going on after trying to mess around with the height values.
Haven't poked around the code yet, but this monaco issue seams to be about what is wanted to happen here: https://github.com/microsoft/monaco-editor/issues/2880  
It mentions `code.split('\n').length * 18` and set that as the height of the monaco-editor.

**Comment by jsoref** [comment]
This seems useful:
```ts
monaco.editor.updateOptions({ scrollBeyondLastLine: false });
```
